lib/trento_web/openapi/v1/schema/platform.ex

```diff
@@ -13,10 +13,6 @@ defmodule TrentoWeb.OpenApi.V1.Schema.Platform do
       type: :object,
       additionalProperties: false,
       properties: %{
-        eula_accepted: %Schema{
```

Comment:
Removing this field breaks API backward compatibility. We need to add a v2 version of it.